.github/ISSUE_TEMPLATE/new-package.md

```diff
@@ -0,0 +1,28 @@
+---
+name: New package
+about: Suggest software for inclusion in the repo
+title: Package <name>
+labels: package
+assignees: ''
+
+---
+
+<!--
+Please fill out all the fields below if possible!
+Make sure the software
+1. does not already exist in the official Arch Linux repos: https://archlinux.org/packages/
+2. does not already exist in this repo: https://arch.osamc.de/#packages
+3. is released under a free software license
+-->
+
+- Project name: 
+- Suggested package name: 
+- Website: 
+- Source code: 
+- Existing AUR: https://aur.archlinux.org/packages/name
```

Comment:
Add `- Current release:`.

Sometimes this is actually hard to find out :)

Reply:
93e66cbdd6dd6428d27e81344a699d361c5da285